config/schema.yml

```diff
@@ -105,7 +105,7 @@ propel:
     name: { type: varchar(1024), required: true }
     path: { type: varchar(1024), required: true }
     sequence: integer
-    byte_size: integer
+    byte_size: bigint
```

Comment:
Could we do the same for https://github.com/artefactual/atom/blob/ad6943d07d99e614fe2636e312c308ec39fefaee/config/schema.yml#L285

Reply:
Ah, good call. I'll make that tweak.